test/bats.bats

```diff
@@ -656,8 +650,7 @@ END_OF_ERR_MSG
 }
```

Comment:
```suggestion
  cp "$BATS_TEST_TMPDIR/tab in filename.bats" "$FIXTURE_ROOT/tab"$'\t'"in filename.bats"
  bats "$BATS_TEST_TMPDIR/tab"$'\t'"in filename.bats"
}
```

I meant to change this, which is very similar to your change below and profits from your newly added variable.